frontend/src/pages/SignupPage/index.module.css

```diff
@@ -1,21 +1,28 @@
-.root {
+.main {
   display: flex;
   flex-direction: column;
+  flex-grow: 1;
   align-items: center;
-  min-height: 100vh;
-  padding-bottom: 60px;
 }
 
-.main {
+.component {
   display: flex;
   flex-direction: column;
   flex-grow: 1;
   align-items: center;
-  padding: 0 var(--main-container-spacing);
 }
 
 .logo {
   display: flex;
   justify-content: flex-end;
-  padding: 50px;
+  margin-bottom: 45px;
+}
+
+.isHidden {
+  visibility: hidden;
```

Comment:
Do we want to just toggle the visibility here? I mean, do we want to just hide it but keep it in the DOM, accessible for screen readers and taking space in the document, or do we want to remove it completely?

Reply:
I think we want to just hide it and 'reserve' that space

Reply:
Cool then it's good to go 👌